manifests/profile/prometheus/exporter/node.pp

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2019 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+class nebula::profile::prometheus::exporter::node (
+  Array $covered_datacenters = [],
+  String $default_datacenter = 'default',
+) {
+  case lookup('role') {
```

Comment:
Would this make sense to extract as a function?

Reply:
Yes, and done.